tests/jerry/es.next/function-await1.js

```diff
@@ -51,11 +55,7 @@ async (a) => {
 }
 
 (a) => {
-  await
   async (a) => await a
-  await
-  async (a) => await a
-  a\u0077ait
 }
```

Comment:
Why these needs to be changed? FF and Chrome accepts them.

Reply:
I followed node's behaviour, and node only allows `await` inside async functions. What I overlooked is that node will only error if the `await` actually has an argument, but if `await` stands by itself then it is an identifier, for example:
```
function f(a) {
  await; // OK in node (sees it as an identifier), SyntaxError in jerry
  await a; // SyntaxError in node and jerry
}
```
But V8 and FF have implemented the top level await proposal, which we have not, so we have to deal with this somehow. I think it's okay to now allow await anywhere outside of async until we implement the proposal too.